examples/Resistor_Tutorial/resistor_tutorial.md

```diff
@@ -0,0 +1,295 @@
+from openhtf import measures, Measurement
+
+@measures(
+        Measurement("resistor_val")
+        .doc("Computed resistor value")
+        )
+def resistor_test(test):
+    pass
+
+def main():
+    test = Test(resistor_test)
+    test.add_output_callbacks(console_summary.ConsoleSummary())
+    test.execute()
+
+if __name__ == "__main__":
+    main()
+```
+
+This gives us a more detailed output which is very helpful for debugging when developing tests : 
+```shell
+(.venv-htf)$ python main_test.py
+:FAIL
+failed phase: resistor_test [ran for 0.00 sec]
+  failed_item: resistor_val (Outcome.UNSET)
+    measured_value: UNSET
+    validators:
+
+
+======================= test: openhtf_test  outcome: FAIL ======================
+```
+
+To define the measurement, we just need to add the line : `test.measurements["resistor_val"] = 10` inside of our `resistor_test` phase. We set it as an arbitrary value for now, but soon this will be given by an actual measurement.
+
+```python
+@measures(
+        Measurement("resistor_val")
+        .doc("Computed resistor value")
+        )
+def resistor_test(test):
+    test.measurements["resistor_val"] = 10
+
+```
+
+## Creating Plugs
+
+To move on with our test definition, we need to be able to communicate with lab instruments. In our case, this is going to be a multimeter and a power supply. OpenHTF is meant to be very modular so the boilerplate code that allows us to interact with each device is going to be implemented inside of a `Plug` object in a seperate script. All we need to do for testing is to import the Plugs we need within our test script, and then interact with the device using the Plugs API. 
+
+The goal is not to dive into how to develop Plugs, so we'll just provide the code for the multimeter and the power supply. The tutorial was developped with a Rigol DM858 and Rigol DP932E, but it also includes a simulation mode so you can run everything without the hardware. 
+
+
+To use your plugs in a test phase, import them in your script and then declare them inside of a decorator for your test phase : 
+
+```python
+from openhtf import plug
+from my_plugs import MultimeterPlug, PowerSupplyPlug
+
+import time
+```
+and then 
+```python
+@measures(
+        Measurement("resistor_val")
+        .doc("Computed resistor value")
+        )
+@plug(dmm=MultimeterPlug)
+@plug(supply=PowerSupplyPlug)
+def resistor_test(test, dmm:MultimeterPlug, supply:PowerSupplyPlug) -> None:
+    test.measurements["resistor_val"] = 10
+```
+
+All that's left to do is to connect to the instruments, set the voltage, read the resulting current through the circuit and calculate the resistor value: 
+
+```python
+@measures(
+        Measurement("resistor_val")
+        .doc("Computed resistor value")
+        )
+@plug(dmm=MultimeterPlug)
+@plug(supply=PowerSupplyPlug)
+def resistor_test(test, dmm:MultimeterPlug, supply:PowerSupplyPlug) -> None:
+    supply.connect()
+    dmm.connect()
+    
+    input_voltage = 5 #in V
```

Comment:
Openhtf has the `util.units` module, do you want to introduce that here, instead of commenting that this is a voltage?